picoquicfirst/picoquicdemo.c

```diff
@@ -899,6 +925,8 @@ int quic_client(const char* ip_address_text, int server_port, const char * sni,
 
             PICOQUIC_SET_LOG(qclient, F_log);
 
+            picoquic_set_key_log_file_from_env(qclient);
+
             if (sni == NULL) {
```

Comment:
I wonder whether the client should display some text "Keys are logged on file %s", so the operators knows what is happening.